src/main/java/org/jabref/gui/maintable/BibEntryTableViewModel.java

```diff
@@ -115,28 +115,27 @@ public BibEntry getEntry() {
         ObservableValue<String> value = fieldValues.get(fields);
         if (value != null) {
             return value;
-        } else {
-            value = Bindings.createStringBinding(() -> {
-                boolean isName = false;
-
-                Optional<String> content = Optional.empty();
-                for (Field field : fields) {
-                    content = entry.getResolvedFieldOrAliasLatexFree(field, database);
-                    if (content.isPresent()) {
-                        isName = field.getProperties().contains(FieldProperty.PERSON_NAMES);
-                        break;
-                    }
-                }
+        }
 
-                String result = content.orElse(null);
-                if (isName) {
-                    return nameFormatter.formatName(result);
-                } else {
-                    return result;
+        value = Bindings.createStringBinding(() -> {
+            boolean isName = false;
+
+            Optional<String> content = Optional.empty();
+            for (Field field : fields) {
+                content = entry.getResolvedFieldOrAlias(field, database);
```

Comment:
This now has the problem that latex code in other fields such as the title is no longer resolved. I think the `isName` check needs to be used before getting the field content.

Reply:
> This now has the problem that latex code in other fields such as the title is no longer resolved.

That is not great :P 
I am unable to find where this happens, if you can provide me with an example I can hopefully fix it? I'd guess there is a cached value somewhere that I have missed.
Assuming it is only a caching mistake I should be able to improve the readability before this is merged into the master.

My thoughts regarding the changes were that,
The previous behaviour of these lines were to return a resolved and latex-free name (by the `isName` check) or null. The current behaviour (should be?) to return a resolved and latex-free name or an empty string (""), with lines 133-135 performing the same functionality as the previous `isName` check.

Reply:
Scratch that, I found it.

Reply:
Hum... I ran into an issue similar to #3356. I'll see if I can fix it/open up an issue on it later.